epitator/count_annotator.py

```diff
@@ -200,8 +181,55 @@ def search_regex(regex_term, match_name=None):
         annotated_counts = ra.combine(
             [single_sentence_counts], prefer='num_spans')
 
-        return {
-            'counts': AnnoTier([
-                CountSpan(count)
-                for count in annotated_counts
-            ])}
+        attributes = [
```

Comment:
We seem to be mixing single and double quotes - we should probably just pick one and stick with it?